user-operator/src/main/java/io/strimzi/operator/user/model/KafkaUserModel.java

```diff
@@ -133,13 +134,23 @@ public void maybeGenerateCertificates(CertManager certManager,
         } else if (clientsCaKeySecret == null) {
             throw new NoCertificateSecretException("The Clients CA Key Secret is missing");
         } else {
+            String valStr = System.getenv(ENV_VAR_CLIENTS_CA_VALIDITY);
```

Comment:
You should not be getting the env vars here but rather in `UserOperatorConfig` where we take and process other env vars as well. That will allow us to have them all in one place.